tools/onnx_subgraph/src/lib/device.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2025 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "device.h"
+
+void Device::GetDeviceJson(std::string json_path) { return; }
```

Comment:
```suggestion
void Device::GetDeviceJson(const std::string json_path) { return; }
```
or
void Device::GetDeviceJson(const std::string &json_path) { return; }